etc/cget/cmake/autotools-build.cmake

```diff
@@ -0,0 +1,151 @@
+##############################################################################

```

Comment:
please add an eclipse copyright header. You can have a look on CMake files from 4diac FORTE for examples.